src/RemoteStorageService/Confirmation.test.js

```diff
@@ -0,0 +1,51 @@
+import { render } from '@testing-library/react';
+import { Heading, Message } from './Confirmation';
+
+jest.mock('react-intl', () => {
```

Comment:
@KetineniM I wonder if you could avoid mocking react-intl if you wrap Heading and Message in `renderWithIntl`

Reply:
@mkuklis - updated, please review and let me know if any changes are required